options/options.go

```diff
@@ -205,6 +206,29 @@ func boolSum(bools ...bool) int {
 
 // Feature flags.
 
+// GitHub integration support.
+// See https://github.com/ossf/scorecard-action/issues/1107.
+// NOTE: We don't add a field to to the Option structure to simplify
+// integration. If we did, the Action would also need to be aware
+// of the integration and pass the relevant values. This
+// would add redundancy and complicate maintenance.
+func (o *Options) IsInternalGitHubIntegrationEnabled() bool {
+	return (os.Getenv("CI") == "true") &&
+		(os.Getenv("SCORECARD_INTERNAL_GITHUB_INTEGRATION") == "1") &&
+		(os.Getenv("GITHUB_EVENT_NAME") == "<todo>")
```

Comment:
@anuraag016 what is the name of the event?

Reply:
its "dynamic"

Reply:
Thanks. Updated.